src/app/timeseries-menu.ts

```diff
@@ -78,6 +79,23 @@ export class TimeseriesMenu {
       }
     }
   }
+  closeTimeSeries() {
+    this.dialogRef.close();
+  }
+  yValTitle(): String {
```

Comment:
You did this same logic somewhere else (for the title fixing). You should pull this logic out and put it in the same function that both these parts of the code call.